src/main/java/org/folio/gobi/DataSourceResolver.java

```diff
@@ -45,9 +45,9 @@ public static Builder builder() {
     CompletableFuture<Object> future = new CompletableFuture<>();
 
     if (from != null) {
-      applyXPath(doc)
-        .thenAccept(s -> applyTranslation(s)
-          .thenAccept(o -> applyDefault(o, doc)
+      CompletableFuture.completedStage(applyXPath(doc)
```

Comment:
Also what is the reason to add CompletableFuture.completedStage() here?
applyXPath(doc) already returned CompletableFuture